<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="storeButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Calibri Light</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="accessibleName">
            <string/>
           </property>
           <property name="text">
            <string>Store</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="profileButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Calibri Light</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>My profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="statisticButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Calibri Light</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Statistics</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="outButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Calibri Light</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Sign out</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <widget class="QGroupBox" name="statisticField">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>451</width>
         <height>331</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri Light</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Statistics:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
       <widget class="QLabel" name="results">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>40</y>
          <width>261</width>
          <height>211</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
